daemon/health.go

```diff
@@ -136,7 +138,9 @@ func handleProbeResult(d *Daemon, c *container.Container, result *types.Healthch
 	}
 
 	if oldStatus != h.Status {
-		d.LogContainerEvent(c, "health_status: "+h.Status)
+		d.LogContainerEventWithAttributes(c, "health_status", map[string]string{
```

Comment:
This is an API change. We'll have to update consumers in swarmkit that take health events.